exporters/autoexport/signal.go

```diff
@@ -51,8 +51,8 @@ func (s signal[T]) create(ctx context.Context, opts ...option[T]) (T, error) {
 }
 
 type config[T any] struct {
-	hasFallback bool
-	fallback    T
+	hasFallback     bool
```

Comment:
Can we get rid of `hasFallback` and simply use `if fallbackFactory != nil` instead?

Reply:
Yes, you are right it seems we can get rid of "hasFallback" 👍